frontends/web/src/locales/en/app.json

```diff
@@ -99,6 +99,8 @@
   },
   "auth": {
     "authButton": "Authenticate",
+    "dismissButton": "Cancel",
+    "missing": "Please, setup an authentication method on your device to continue",
```

Comment:
no comma after `Please` please :smile: 

Maybe better: "Please setup a screen lock in your device settings".